lib/shared/addon/components/form-ingress-backends/template.hbs

```diff
@@ -65,14 +68,19 @@
 
           <td class="p-5" data-title="{{t 'formIngressBackends.targetPort.label'}}">
             {{#input-or-display editable=editing value=path.targetPort classesForDisplay="clip"}}
-              {{#if (and (eq path.backendType 'service') (eq path.service.recordType 'workload'))}}
+              {{#if (and path.service (eq path.backendType 'service'))}}
                 {{new-select
                   classNames="form-control input-sm"
                   optionValuePath="port"
                   optionLabelPath="port"
                   content=path.service.availablePorts
                   value=path.targetPort
                 }}
+              {{else if (eq path.backendType 'service')}}
+                {{new-select
```

Comment:
I'm not sure what the point of an empty select is here; if the path entry is valid without a targetPort then just don't have a select.  If it's not then disable the service entry so I can't select it in the first place?